ee/packages/amplication-git-pull-service/src/git-pull-event/git-host-provider.service.ts

```diff
@@ -1,18 +1,17 @@
 import { Injectable } from "@nestjs/common";
-import { IGitProvider } from "../../contracts/interfaces/gitProvider.interface";
+import { GitProvider } from "./git-pull-event.types";
 import { App } from "octokit";
-import { convertToNumber } from "../../utils/convertToNumber";
+import { convertToNumber } from "../utils/convert-to-number";
 import { ConfigService } from "@nestjs/config";
-import { CustomError } from "../../errors/CustomError";
-import { ErrorMessages } from "../../constants/errorMessages";
+import { ErrorMessages } from "./git-pull-event.constants";
 
 const GITHUB_APP_APP_ID_VAR = "GITHUB_APP_APP_ID";
 const GITHUB_APP_PRIVATE_KEY_VAR = "GITHUB_APP_PRIVATE_KEY";
 /*
  * Octokit integration
  * */
 @Injectable()
-export class GitHostProviderService implements IGitProvider {
```

Comment:
Let's get rid of this convention, it's redundant and uncommon. Let's also enforce this in other codebases.